content/modules/testing/pages/index.adoc

```diff
@@ -0,0 +1,9 @@
+= Testing
+
+Automated Testing is a core part of Jmix as it allows creating long-term maintainable applications. Jmix provides various capabilities to support the application developer to create different types of automated tests.
+
+Since Jmix is based on Spring Boot, all Spring Boot Testing capabilities also apply to Jmix.
+
+Spring has comprehensive documentation in their https://docs.spring.io/spring-framework/docs/current/reference/html/testing.html[Spring Framework: Testing reference documentation] as well as in Spring Boot's https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.testing[Testing feature documentation].
```

Comment:
The link https://docs.spring.io/spring-framework/docs/current/reference/html/testing.html doesn't go where intended.

Also, add ^ to all external links to open in a new tab, e.g. https://address[Title^].

Reply:
done